Taskfile.yml

```
@@ -364,16 +370,19 @@ tasks:
           DATA_DIR: "{{.OUTPUT_DIR}}"
           OUTPUT_FILE: "{{.CHECKSUM_FILE}}"
 
-  # NOTE: The log-viewer-webui has three different node_modules directories (client, server, and the
-  # top-level one we call "package"), meaning we have to create three different checksums. To allow
-  # tasks which depend on this task to only have to check one checksum file, we concatenate the
-  # three checksum files into one.
+  # NOTE: The log-viewer-webui has four different node_modules directories
+  # (client, server, log-viewer submodule, and the top-level one we call "package"),
+  # meaning we have to create four different checksums.
+  # To allow tasks which depend on this task to only have to check one checksum file,
+  # we concatenate the three checksum files into one.
```

```suggestion
  # NOTE: The log-viewer-webui has four different node_modules directories:
  # * client
  # * server
  # * log-viewer submodule
  # * the top-level one we call "package"
  # This means we have to create four different checksums. To allow tasks which depend on this task
  # to only have to check one checksum file, we concatenate the four checksum files into one.
```

Let's try to format comments with Markdown formatting so that in the future, we can easily automate formatting comments.